**Question 1** Calculate the probability of flipping a balanced coin four times and getting each pattern: HTTH, HHHH and TTHH.

P(HTTH) = P(HHHH) = P(TTHH) = (.5) * (.5) * (.5) * (.5) = 0.0625


**Question 2** If a list of people has 24 women and 21 men, then the probability of choosing a man from the list is 21/45. What is the probability of not choosing a man?

In [3]:
p = 1 - (21/45)
p

0.5333333333333333

P(not Man) = 1 - P(Man) = 1 - (21/45) =  0.5333333333333333

**Question 3** The probability that Bernice will travel by plane sometime in the next year is 10%. The probability of a plane crash at any time is .005%. What is the probability that Bernice will be in a plane crash sometime in the next year?

P(Crash) = .005/100

P(Travel) = 10/100

P(Crash n Travel) = P(Crash) * P(Travel) = (.005/100) * (10/100)

In [7]:
p = (.005/100)*(10/100)
p

5e-06

**Question 4** A data scientist wants to study the behavior of users on the company website. Each time a user clicks on a link on the website, there is a 5% chance that the user will be asked to complete a short survey about their behavior on the website. The data scientist uses the survey data to conclude that, on average, users spend 15 minutes surfing the company website before moving on to other things. What is wrong with this conclusion?

The study assumes that people who answered the survey are representative of the entire population. Users who click more links are more likely to recieve a survey. There may be users who only click one link to go to a certain part of the website and spend an unknown amount of time there which are less likely to be asked to fill out the survey that users who spend their time bouncing between different parts of the website.  

**------- Drill Set 2 -------** 

98% probability of giving true positive

10% probability of giving false positive

0.5% of population are thripshaw disease sufferers


**Question 1:** Calculate the probability that the test will be positive

In [19]:
prob_positive = (0.5/100)*(.98) + (1-(0.5/100))*(.1) 
prob_positive

0.1044

**Question 2** Calculate probability to correctly diagnose sufferer of thripshaws

**NOTE:** For this question the solutions say .98. The way I interpreted this problem is if a doctor administer's the test to a random person, what is the probability that that person is diagnosed with thripshaws? This is the way it seems the question is phrased. The doctor does not have a 98% chance of correctly diagnosing someone who walks into their office unless we are told the person already has thripshaws.  

In [25]:
prob_true_positive = (0.5/100)*(.98)
prob_true_positive

0.0049

**Question 3** Calculate probabilty to correctly identify non-sufferer of Thripshaws

In [21]:
prob_true_negative = (1-(0.5/100))*(1-.1)
prob_true_negative

0.8955

**Question 4** Calculate probability to misclassify the person

In [23]:
prob_false_positive = (0.5/100)*(2/100)
prob_false_negative = (1-(0.5/100))*(10/100)
prob_misclassified = prob_false_positive + prob_false_negative
prob_misclassified

0.09960000000000001